fsharp-backend/src/ApiServer/Ui.fs

```diff
@@ -152,11 +153,19 @@ let uiHandler (ctx : HttpContext) : Task<string> =
     if integrationTests && Config.allowTestRoutes then
       do! LibBackend.Canvas.loadAndResaveFromTestFile canvasInfo
 
+    let! isCanvasOwnerAdmin =
```

Comment:
Can you rename this to something like `canAccessOperations`?

This isn't really whether the canvas owner is an admin, it's whether we can use operations functions on this canvas.